package.json

```diff
@@ -11,6 +11,7 @@
     "meson-setup.clang-debug": "meson setup --reconfigure build/ --buildtype=debug --native-file toolchains/clang.ini",
     "meson-setup.zig-release": "meson setup --reconfigure build/ --buildtype=release --native-file toolchains/zig.ini",
     "vs.example": "meson compile -C build/ vs:executable &&  VS_COMMONS_DIR=$(pwd)/build/commons/ ./build/vs run vs://examples/example-0.xml",
+    "vs.devenv": "meson devenv -C build",
```

Comment:
I think `meson devenv` is intended to be run from the build directory. One problem that happens using this PR is that after entering the devenv, you are left in the build directory so therefore 'bun run...' doesn't work.

I understand why a person is left in the build directory... the '-C build' does the change.